cmd/controller-bootstrap/command/update.go

```diff
@@ -0,0 +1,81 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package command
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/spf13/cobra"
+)
+
+var projectDescriptionFiles = []string{
+	"CODE_OF_CONDUCT.md.tpl",
+	"CONTRIBUTING.md.tpl",
+	"GOVERNANCE.md.tpl",
+	"LICENSE.tpl",
+	"NOTICE.tpl",
+	"READ_BEFORE_COMMIT.md.tpl",
```

Comment:
Not sure if this file needs to be updated in source controller repositories. This is the file that gets output when the controller repository is initially generated, right? To tell the generating user what to do next... I don't think it gets copied to the controller repository does it?

Reply:
Yes, this is the file that gets output when the controller repository is initially generated but it will not be copied to the controller repository. This has been fixed, thank you.